build.sbt

```diff
@@ -1,49 +1,78 @@
-inThisBuild(List(
-  organization := "com.dwolla",
-  description := "Adds support for decrypting values in TypeSafe Config files",
-  homepage := Some(url("https://github.com/Dwolla/scala-secure-config")),
-  licenses += ("MIT", url("http://opensource.org/licenses/MIT")),
-  developers := List(
-    Developer(
-      "bpholt",
-      "Brian Holt",
-      "bholt+github@dwolla.com",
-      url("https://dwolla.com")
-    ),
+import org.typelevel.sbt.gha.WorkflowStep.Sbt
+
+ThisBuild / organization := "com.dwolla"
+ThisBuild / description := "Adds support for decrypting values in TypeSafe Config files"
+ThisBuild / homepage := Some(url("https://github.com/Dwolla/scala-secure-config"))
+ThisBuild / licenses += ("MIT", url("http://opensource.org/licenses/MIT"))
+ThisBuild / developers := List(
+  Developer(
+    "bpholt",
+    "Brian Holt",
+    "bholt+github@dwolla.com",
+    url("https://dwolla.com")
   ),
-  crossScalaVersions := Seq("2.13.14", "2.12.20"),
-  scalaVersion := crossScalaVersions.value.head,
-  startYear := Option(2018),
-  addCompilerPlugin("org.typelevel" %% "kind-projector" % "0.13.3" cross CrossVersion.full),
-  addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1"),
+)
+ThisBuild / crossScalaVersions := Seq(
+  "2.13.14",
+  "3.3.3",
+)
+ThisBuild / scalaVersion := crossScalaVersions.value.head
+ThisBuild / startYear := Option(2018)
+ThisBuild / tlBaseVersion := "0.4"
+ThisBuild / tlJdkRelease := Some(8)
+ThisBuild / githubWorkflowBuild := List(Sbt(List("compile", "test")))
```

Comment:
Adding the extra `compile` step here ensures that sbt recompiles the code after it's modified by Scalafix. When the `test` task is executed after the explicit `compile` step, sbt will see that the code has changed since the `compile` step and recompile it.